test/localstack.ts

```diff
@@ -42,15 +44,10 @@ export const deployStack = async (app: App, stack: Stack) => {
     sdkProvider,
   });
 
-  const assembly = new cxapi.CloudAssembly(cloudAssembly.directory);
-  const stackArtifact = cxapi.CloudFormationStackArtifact.fromManifest(
-    assembly,
-    stack.artifactId,
-    cloudAssembly.getStackArtifact(stack.artifactId).manifest
-  ) as cxapi.CloudFormationStackArtifact;
-
   await cfn.deployStack({
```

Comment:
right but presumably we only do this to meet localstacks expected CDK 1.x input?

Reply:
Ohh, no, cdk is running locally and localstack only sees the cfn. This is all because CDK doesn't support (aka makes it hard as fuck) to synth and deploy programmatically.